Feature/mock ssdt txt tc

Ellen99999999 opened the pull request:
[Feature] ssd mock tc 동작 보강

변경점 : ssd mock에 실제로 파일 읽고 쓰기 형식으로 tc 를 변경합니다.

검증

[ v] Build

[v ] Regression Test Pass

[ v] 해당 기능 TC

Ellen99999999 committed:
[Feature] SSD Mock TC update

변경점 : ssd mock 에서 실제 파일을 읽고 쓰고 체크하도록 변경함

검증

[v ] Build

[v ] Regression Test Pass

[v ] 해당 기능 TC

Ellen99999999 committed:
[Feature] SSD Mock TC update

변경점 : ssd mock 에서 실제 파일을 읽고 쓰고 체크하도록 변경함

검증

[v ] Build

[v ] Regression Test Pass

[v ] 해당 기능 TC

Ellen99999999 committed:
Merge branch 'feature/mock_ssdt_txt_tc' of https://github.com/flowinto10/CRA_TeamBest_SSDShell into feature/mock_ssdt_txt_tc

flowinto10 commented on TeamBest_SSD_Shell/test_ssd.cpp:
```diff
@@ -1,23 +1,64 @@
+		return true;
+	}
+
+	string readFromFile(const string& filename) {
```
Shell 객체에서 해야할 일을 아직 객체가 없어서 여기에서 하신걸까요?

Ellen99999999 replied:
네네 아직 shell 도 ssd 도 완성이 안되서요

Ellen99999999 committed:
[Feature] tc 보강(SSD Mock)

변경점 : 파일이 없거나 하는 경우 fail 로 판정하도록 수정함

검증

[v ] Build

[v ] Regression Test Pass

[v ] 해당 기능 TC

flowinto10 reviewed (approved):
넵 고생하셨습니다.

youngchul-hwang reviewed (approved):
Looks Good To Me!
고생하셨습니다.

R1NRINRIN reviewed (approved):
좋은 코드입니다. 고생하셨습니다.
